docs/source/using/managing-minishift.adoc

```diff
@@ -53,6 +53,41 @@ the GitHub issue https://github.com/minishift/minishift/issues/179[#179].
 The link:../command-ref/minishift_delete{outfilesuffix}[`minishift delete`] command deletes the OpenShift cluster,
 and also shuts down and deletes the Minishift VM. No data or state are preserved.
 
+[[choosing-iso-image-for-vm]]
+== Choosing the ISO image for the VM
+
+When you start Minishift, it downloads a live ISO image that the hypervisor uses to provision the host VM.
+This VM then locally provisions the OpenShift cluster.
+
+The following ISO images are available:
+
+- link:https://github.com/minishift/minishift-b2d-iso/releases[Boot2Docker] (Default). This ISO is
+based on the link:https://github.com/boot2docker/boot2docker[Boot2Docker] ISO, which is a
+lightweight Linux distribution optimized for containerized application development.
+
+- link:https://github.com/minishift/minishift-centos-iso/releases[CentOS]. This ISO contains
+the link:https://www.centos.org/[CentOS] operating system, which is an enterprise-ready Linux
+distribution.
++
+NOTE: The CentOS ISO image size is larger than the Boot2Docker image, because CentOS is a
+full-featured enterprise Linux operating system.
+
+By default, Minishift uses the **Boot2Docker** image. To choose the CentOS ISO instead,
+you can do one of the following actions:
+
+- Set the `--iso-url` flag when you run `minishift start` and enter the download URL of
+the CentOS ISO in the following format:
++
+----
+$ minishift start --iso-url <Web_location_of_ISO_image>
```

Comment:
Because when discussing this with @LalatenduMohanty and @gbraad on IRC it was determined that maintaining an example URL through the different ISO versions would be cumbersome (i.e. to make sure that it points to the correct release). Therefore, I added a link to the releases page where users can grab the latest URL, and used the format instead. Lala approved this step.